recipe/meta.yaml

```diff
@@ -33,6 +35,9 @@ requirements:
     - mkl-service
```

Comment:
This is also optional.

Reply:
Do you mean that numpy and pandas are optional, or something about the pinning is optional?

Reply:
`mkl-service` is not a requirement.

Reply:
Ah, sorry, it seems that GitHub glitched and was showing pandas/numpy instead of mkl-service.